mobile/src/profile/styles.ts

```diff
@@ -138,10 +142,13 @@ export const tileStyles = StyleSheet.create({
 export const contactStyles = StyleSheet.create({
     container: {
         flexDirection: 'row',
-        paddingBottom: 10
+       paddingLeft: 45,
```

Comment:
Are you trying to align contacts with padding?

Reply:
I align left contacts, why not?

Reply:
Because it looks like centered
![capture](https://user-images.githubusercontent.com/783407/36414404-3b5ff78a-1633-11e8-9967-61e4a922e9e6.PNG)

Reply:
Our designer (Ilya) said it should be left align.